var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -95,6 +99,12 @@ class PyScipy(PythonPackage):
 
     patch('scipy-clang.patch', when='@1.5.0:1.6.3 %clang')
 
+    @run_before('install')
+    def set_blas_lapack(self):
+        # Pick up Blas/Lapack from numpy
+        numpy = PyNumpy(self.spec['py-numpy'])
+        numpy.set_blas_lapack()
```

Comment:
Is this equivalent to `self.spec['py-numpy'].package.set_blas_lapack()`? Same below, then we wouldn't need the import.

Reply:
So it is! I have fixed that